pkg/controllers/scheduler/framework/plugins/clusterresources/least_allocated.go

```diff
@@ -49,21 +49,22 @@ func (pl *ClusterResourcesLeastAllocated) Score(
 		return 0, framework.NewResult(framework.Error, err.Error())
 	}
 
-	requested := make(framework.ResourceToValueMap, len(framework.DefaultRequestedRatioResources))
-	allocatable := make(framework.ResourceToValueMap, len(framework.DefaultRequestedRatioResources))
-	for resource := range framework.DefaultRequestedRatioResources {
-		allocatable[resource], requested[resource] = calculateResourceAllocatableRequest(su, cluster, resource)
-	}
+	resources := getRelevantResources(su)
+	allocatable, requested := getAllocatableAndRequested(su, cluster, resources)
 
 	var score, weightSum int64
 	// least allocated score favors cluster with fewer requested resources.
 	// It calculates the percentage of memory and CPU requested by pods scheduled on the cluster, and
 	// prioritizes based on the minimum of the average of the fraction of requested to capacity.
 	//
 	// Details:
-	// (cpu((capacity-sum(requested))*10/capacity) + memory((capacity-sum(requested))*10/capacity))/2
-	for resource, weight := range framework.DefaultRequestedRatioResources {
+	// (cpu((capacity-sum(requested))*100/capacity) + memory((capacity-sum(requested))*100/capacity))/2
```

Comment:
This example assumes weight ratios of 1:1:4. It's possible that we might change the ratio in the code and forget to update these comments. Should we make it clear here that we're assuming 1:1:4?

Reply:
How about not using 4 but using `gpu weight` in comments?